src/components/organisms/NewProfileModal/components/buttons/UserProfileModalButtons/UserProfileModalButtons.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+import {
+  profileModalWideButtonCustomStyle,
+  profileModalWideButtonCustomStyleDisabled,
+  profileModalWideButtonCustomStyleGrey,
+} from "types/profileModal";
+import { ContainerClassName } from "types/utility";
+
+import { Button } from "components/atoms/Button";
+
+import "./UserProfileModalButtons.scss";
+
+export interface UserProfileModalButtonsProps extends ContainerClassName {
+  onCancelClick: () => void;
+  isSubmitting: boolean;
+}
+
+export const UserProfileModalButtons: React.FC<UserProfileModalButtonsProps> = ({
+  onCancelClick,
+  containerClassName,
+  isSubmitting,
+}) => {
+  return (
+    <div className={containerClassName}>
+      <Button
+        customClass={"UserProfileModalButtons__button"}
+        customStyle={
+          isSubmitting
+            ? profileModalWideButtonCustomStyleDisabled
+            : profileModalWideButtonCustomStyleGrey
+        }
```

Comment:
just use classnames

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1934/commits/9a89efc5877a94e36edd372aba8c28c8dc82304d